front-end/src/views/management/roleBinding/index.vue

```diff
@@ -0,0 +1,328 @@
+          <el-input :rows="2" v-model="form.description" placeholder="Please input description" type="textarea"/>
+        </el-form-item>
+        <el-form-item v-if="dialogStatus==='delete'">
+          <h4>Delete Role Binding {{ form.name }}</h4>
+        </el-form-item>
+        <el-form-item>
+          <el-button type="primary" @click="handleOptions()">{{ $t('table.confirm') }}</el-button>
+          <el-button @click="dialogFormVisible = false">{{ $t('table.cancel') }}</el-button>
+        </el-form-item>
+      </el-form>
+    </el-dialog>
+  </div>
+</template>
+
+<script>
+import {
+  fetchRoleBinding,
+  putRoleBinding,
+  updateRoleBinding,
+  deleteRoleBinding
+} from '@/api/roleBinding'
+
+import { fetchRoles } from '@/api/roles'
+
+export default {
+  name: 'RoleBinding',
+  data() {
+    return {
+      roleBindingList: [],
+      roleBindingTableKey: 0,
+      roleBindingListLoading: false,
+      textMap: {
+        create: 'New role binding',
+        delete: 'Delete role binding',
+        update: 'Update role binding'
+      },
+      dialogFormVisible: false,
+      dialogStatus: '',
+      form: {
+        name: '',
+        description: '',
+        roleName: '',
+        roleId: 0,
+        userName: '',
+        userId: 0
+      },
+      roleListOptions: [],
+      roleNameValue: '',
+      temp: {
+        'name': '',
+        'description': '',
+        'roleName': '',
+        'roleId': 0,
+        'userName': '',
+        'userId': 0
+      },
+      superUser: false,
+      description: '',
+      rules: {
+        name: [{ required: true, message: 'Name is required', trigger: 'blur' }]
+      }
+    }
+  },
+  created() {
+    this.getRoleBinding()
+  },
+  methods: {
+    getRoleBinding() {
+      fetchRoleBinding().then(response => {
+        if (!response.data) return
+        this.roleBindingList = []
+        for (var i = 0; i < response.data.data.length; i++) {
+          this.roleBindingList.push({
+            'name': response.data.data[i].name,
+            'description': response.data.data[i].description,
+            'roleName': response.data.data[i].roleName,
+            'roleId': response.data.data[i].roleId,
+            'userName': response.data.data[i].userName,
+            'userId': response.data.data[i].userId
+          })
+        }
+      })
+    },
+    handleCreateRoleBinding() {
+      this.form.name = ''
+      this.form.description = ''
+      this.form.roleName = ''
+      this.form.roleId = 0
+      this.form.userName = ''
+      this.form.userId = 0
+      this.dialogFormVisible = true
+      this.handleGetRoles()
+      this.dialogStatus = 'create'
+    },
+    handleDeleteRole(row) {
+      this.temp.name = row.name
+      this.temp.roleId = row.roleId
+      this.temp.userId = row.userId
+      this.dialogFormVisible = true
+      this.dialogStatus = 'delete'
+    },
+    handleUpdateRole(row) {
+      console.log(row)
```

Comment:
Nit: remove debug

Reply:
@eolivelli Thank you very much for pointing out the problem.